dist/mapdc.js

```diff
@@ -43033,15 +43047,31 @@ exports.default = bbox;
 
 /***/ }),
 /* 196 */
-/***/ (function(module, exports, __webpack_require__) {
+/***/ (function(module, __webpack_exports__, __webpack_require__) {
```

Comment:
You have the same compilation artifacts as me. I'm pretty sure we are doing something different than @uyanga-gb. It's not a problem, but it would make the diff cleaner to figure it out.

Reply:
hmmm. I was running `npm run watch` which runs `webpack watch`. I wonder if doing `npm run build` outputs something different?